### PR TITLE
Deploy Web App Certificate from Key Vault Secret

### PR DESCRIPTION
**Pull request by akurmi**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@akurmi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (akurmi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-web-app-certificate-from-key-vault/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Deploy a Web App certificate from Key Vault secret and use it for creating SSL binding
```
Comment (singhkay) [inline 101-web-app-certificate-from-key-vault/README.md]: This seems like a pretty involved template and user needs to know a few concepts before doing this. Would you say this is 201 or 301 level?

File: 101-web-app-certificate-from-key-vault/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+   "itemDisplayName":"Web App integration with Key Vault",
+   "description":"Deploy a Web App certificate from Key Vault secret and use it for creating SSL binding",
+   "summary":"Deploy a Web App certificate from Key Vault secret and use it for creating SSL binding",
+   "githubUsername":"akurmi",
+   "dateUpdated":"2016-05-04"
```
Comment (singhkay) [inline 101-web-app-certificate-from-key-vault/metadata.json]: Don't forget to update this on the final commit

File: 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```
Comment (singhkay) [inline 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json]: See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci about placeholder parameters

File: 101-web-app-certificate-from-key-vault/README.md (hunk 1 above)
Comment (akurmi) [inline 101-web-app-certificate-from-key-vault/README.md]: Just moved this to a 201 directory

File: 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json (hunk 3 above)
Comment (akurmi) [inline 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json]: This template assumes that a Web App already exists with a custom hostname assigned to it. I am not sure how changing placeholder parameter values would make it pass the automated PR test.

Comment (singhkay) [inline 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json]: Won't pass but in the future when we can enable CI on existing resources, it'll be one less change to make :)

Comment (akurmi) [inline 101-web-app-certificate-from-key-vault/azuredeploy.parameters.json]: ok, I have updated the parameters file.

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**